tests/conftest.py

```diff
@@ -368,3 +369,20 @@ def _make_function(table_name: str, random_id: str) -> exp.Table:
         return temp_table
 
     return _make_function
+
+
+def create_df(data: t.Sequence[t.Tuple], schema: t.Dict[str, str]) -> pd.DataFrame:
```

Comment:
This is probably minor but I'd suggest reserving `conftest.py` only for fixtures, plugins, hooks and other pytest-specific mechanisms. I'd suggest creating a separate test utils module for this.